MREUnityRuntime/MREUnityRuntimeLib/Behaviors/Contexts/PenBehaviorContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using MixedRealityExtension.Behaviors.ActionData;
+
+namespace MixedRealityExtension.Behaviors.Contexts
+{
+	public class PenBehaviorContext : PhysicalToolBehaviorContext<PenData>
+	{
+		internal PenBehaviorContext()
```

Comment:
Unneeded empty constructor (here and elsewhere).

Reply:
See earlier about accessibility scoping to internal.

---
In reply to: [428858116](https://github.com/microsoft/mixed-reality-extension-unity/pull/160#discussion_r428858116) [](ancestors = 428858116)